common/constants/custom_panels.ts

```diff
@@ -8,7 +8,9 @@ import { v4 as uuidv4 } from 'uuid';
 export const CUSTOM_PANELS_API_PREFIX = '/api/observability/operational_panels';
 export const CUSTOM_PANELS_DOCUMENTATION_URL =
   'https://opensearch.org/docs/latest/observability-plugin/operational-panels/';
-export const CREATE_PANEL_MESSAGE = 'Enter a name to describe the purpose of this Observability Dashboard.';
+export const OPENSEARCH_DOCUMENTATION_URL = 'https://opensearch.org/docs/latest';
```

Comment:
Why this change?
Future of integration is to split it into a separate plugin? - If so, then isolate this constant into a separate file.

Reply:
Moved to new file